### ISSUE TITLE
[Feature] Merge weeks

### ISSUE THREAD

**Issue by aaronjameslang** [AmmsA/Githeat#2]
I'm looking for something showing heat Monday-Sunday 00:00-23:59

Is that a feature you see fitting in here or should I create a seperate project?

**Comment by AmmsA** [comment]
Yes, it fits. If you would like to implement it, go ahead. I'll be happy to review it and merge it.
